src/EprPrnIntegration.Common/Helpers/Utilities.cs

```diff
@@ -23,4 +24,9 @@ public async Task SetDeltaSyncExecution(DeltaSyncExecution syncExecution, DateTi
         syncExecution.LastSyncDateTime = latestRun;
         await serviceBusProvider.SendDeltaSyncExecutionToQueue(syncExecution);
     }
+
```

Comment:
This Utilities class looks like it might become a repository of miscellaneous unrelated stuff

Reply:
I have moved telemetry in utilities as function has already too many injections. Considering utilities can be used for misc things. I will leave it there for now. We can refactor later if required